code-of-conduct/about-code-of-conduct-response-plan.md

```diff
@@ -0,0 +1,182 @@
+* A Code of Conduct response plan template that you can use to create and explain the policy your team will follow as you handle Code of Conduct incidents. This template is included in the same folder as the Code of Conduct template and is named `template-code-of-conduct-response-plan.md`. See its accompany guide: `about-code-of-conduct-response-plan.md`.
+* A Code of Conduct incident record template for a form that is filled out when a community moderator takes an incident report from a community member. This template is included in the same folder as the Code of Conduct template as the Code of Conduct template and is named `template-code-of-conduct-incident-record.md`. See its accompany guide: `about-code-of-conduct-incident-record.md`.
+* A Code of Conduct remediation record template for a form that is filled out when a community moderator meets with a community member to explain the consequences of a Code of Conduct violation. This template is included in the same folder as the Code of Conduct template and is named `template-code-of-conduct-remediation-record.md`. See its accompany guide: `about-code-of-conduct-remediation-record.md`.
+
+You might also consider using the `../our-team/template-our-team.md` template to let your community members know who they can contact to report a Code of Conduct violation. This document is useful beyond Code of Conduct violations. It is a core document that helps you clearly communicate who belongs to your open source project or organization.
+---
+
+
+## Why do I need a Code of Conduct and a response plan?
+
+See the `guide-code-of-conduct.md` for reasons to incorporate a Code of Conduct and response plan.
+
+
+## Content of the Code of Conduct response plan template
+
+The following sections provide guidance about how to fill out each section of the Code of Conduct response plan template.
+
+:information_source: The Code of Conduct response plan template includes boilerplate text that you can customize or adapt, use as-is, or completely replace with your own text. If you use the boilerplate text, make sure you replace the project name placeholders with your own project name.
+
+
+### About the "Introduction" section
+
+The introduction section explains the purpose of the response plan document.
+
+
+### About the "Community moderators" section
+
+In the *Community moderators* section, you can list the names of the community moderators or link to a document that lists the community moderators, such as a website that uses the (Our Team)[../our-team/template-our-team.md] template.
+
+Alternatively, you could list the names of the community moderators and their preferred contact information, such as their Slack handles.
+You could also link to a dedicated email address or contact form.
+
+Remember to respect your community moderator's privacy by only including contact methods that go through official project channels, such as community forums or project-issued email accounts.

```

Comment:
praise: Such an important point to highlight. Keeping personal information private is so important these days